package.json

```diff
@@ -39,8 +39,13 @@
     "webpack-dev-server": "^3.2.1"
   },
   "dependencies": {
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "^2.1.8"
+    "react-redux": "^6.0.1",
+    "react-scripts": "^2.1.8",
+    "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
Great work @wasibani-roy, However, please add redux dev tools as a dev dependency as it will only be used in development.

Reply:
I have done that thanks